lib/http/chainable.rb

```diff
@@ -90,21 +90,18 @@ def build_request(*args)
     # @overload timeout(global_timeout)
     #   Adds a global timeout to the full request
     #   @param [Numeric] global_timeout
+    PER_OPERATION_KEYS = Set.new(%i[read write connect])
```

Comment:
This looks unused :)

Reply:
moved in into lib/http/timeout/per_operation.rb but didn't remove it from here